about/faq/index.html

```diff
@@ -152,7 +151,13 @@ <h3 id="q-how-to-clean-a-dirty-core-folder">Q: How to clean a dirty core folder?
 <h3 id="q-can-i-donate">Q: Can I donate?</h3>
 
 <p>We&rsquo;ve closed donations for now. If you have money burning a hole in your pocket though, consider donating to NotFood. She&rsquo;s
-our headcoder/maintainer. <a href="https://ko-fi.com/notfood">https://ko-fi.com/notfood</a></p>
+our headcoder/maintainer. <a href="https://ko-fi.com/notfood">https://ko-fi.com/notfood</a> or any of the numourous people who've helped out in the past: <br>
+<br>
+<a href="https://ko-fi.com/Nebual">https://ko-fi.com/Nebual</a> (Core) <br>
+<a href="https://ko-fi.com/sokyran">https://ko-fi.com/sokyran</a> (Mod Compatiblity, takes commissions)  <br>
+<a href="https://ko-fi.com/thomas107500">https://ko-fi.com/thomas107500</a> (Mod Compatiblity) <br>
+<a href="https://ko-fi.com/zoeyvs">https://ko-fi.com/zoeyvs</a> (Support/help) <br>
```

Comment:
You definitely should have asked the rest of the team before putting you and mia in this section. Please remove both these links until you do.